### PR TITLE
Add service graph flag

### PR DESCRIPTION
**Pull request by ralongit**
- Updated Helm command
- Updated datasource agent scripts
- Added checkbox to agent ui

### CODE REVIEW

File: configs/2-1-k8s-eks.json [hunk 1]
```diff
@@ -88,7 +88,15 @@
...
               "hint": "Switch the toggle on if you want Logz.io Telemetry Collector to collect traces from your instrumented applications in the Kubernetes environment",
-              "params": []
+              "params": [
+              {
+                "type": "boolean",
+                "name": "isServiceGraph",
+                "label": "Collect Service Graph metrics",
+                "description": "",
+                "hint": "Check this box if you want Logz.io Telemetry Collector to collect Service Graph metrics to populate the Services Overview.",
+                "value": true
```
Comment (mirii1994) [inline configs/2-1-k8s-eks.json:98]: Please verify with @adamcazes that this is ok ux-wise :)

File: configs/2-2-k8s-aks.json [hunk 2]
```diff
@@ -82,7 +82,15 @@
...
-              "params": []
+              "params": [
+              {
+                "type": "boolean",
+                "name": "isServiceGraph",
+                "label": "Collect Service Graph metrics",
+                "description": "",
+                "hint": "Check this box if you want Logz.io Telemetry Collector to collect Service Graph metrics to populate the Services Overview.",
+                "value": true
+              }]
```
Comment (mirii1994) [inline configs/2-2-k8s-aks.json:93]: Please verify with @adamcazes that this is ok ux-wise :)

File: configs/2-3-k8s-gke.json [hunk 3]
```diff
@@ -80,7 +80,15 @@
...
-              "params": []
+              "params": [
+              {
+                "type": "boolean",
+                "name": "isServiceGraph",
+                "label": "Collect Service Graph metrics",
+                "description": "",
+                "hint": "Check this box if you want Logz.io Telemetry Collector to collect Service Graph metrics to populate the Services Overview.",
+                "value": true
+              }]
```
Comment (mirii1994) [inline configs/2-3-k8s-gke.json:91]: Please verify with @adamcazes that this is ok ux-wise :)

File: configs/2-4-k8s-digitalocean.json [hunk 4]
```diff
@@ -80,7 +80,15 @@
...
-              "params": []
+              "params": [
+              {
+                "type": "boolean",
+                "name": "isServiceGraph",
+                "label": "Collect Service Graph metrics",
+                "description": "",
+                "hint": "Check this box if you want Logz.io Telemetry Collector to collect Service Graph metrics to populate the Services Overview.",
+                "value": true
+              }]
```
Comment (mirii1994) [inline configs/2-4-k8s-digitalocean.json:91]: Please verify with @adamcazes that this is ok ux-wise :)

File: datasources/windows/kubernetes/aks/kubernetes/telemetry/traces/traces.ps1 [hunk 5]
```diff
@@ -26,6 +26,11 @@ Invoke-Task 'Get-LogzioSpmToken' @{} 'Getting Logz.io SPM token' @($TracesFuncti
 Invoke-Task 'Build-EnableSpmHelmSet' @{} 'Building enable SPM Helm set' @($TracesFunctionsScript)
 # Build Logz.io SPM token Helm set
 Invoke-Task 'Build-LogzioSpmTokenHelmSet' @{SpmToken = $script:SpmToken} 'Building Logz.io SPM token Helm set' @($TracesFunctionsScript)
-
+# Get if service graph option was selected
+Invoke-Task 'Get-IsServiceGraphWasSelected' @{MetricsParams = $script:MetricsParams} 'Getting if service graph was selected' @($MetricsFunctionsScript)
+if ($script:IsServiceGraph) {
+    # Build enable metrics filter Helm set
```
Comment (mirii1994) [inline datasources/windows/kubernetes/aks/kubernetes/telemetry/traces/traces.ps1:32]: This comment is a copy-paste mistake? it supposed to be "build enable service graph"?

File: datasources/windows/kubernetes/digitalocean/kubernetes/telemetry/traces/traces.ps1 [hunk 6]
```diff
@@ -26,6 +26,11 @@ Invoke-Task 'Get-LogzioSpmToken' @{} 'Getting Logz.io SPM token' @($TracesFuncti
 Invoke-Task 'Build-EnableSpmHelmSet' @{} 'Building enable SPM Helm set' @($TracesFunctionsScript)
 # Build Logz.io SPM token Helm set
 Invoke-Task 'Build-LogzioSpmTokenHelmSet' @{SpmToken = $script:SpmToken} 'Building Logz.io SPM token Helm set' @($TracesFunctionsScript)
-
+# Get if service graph option was selected
+Invoke-Task 'Get-IsServiceGraphWasSelected' @{MetricsParams = $script:MetricsParams} 'Getting if service graph was selected' @($MetricsFunctionsScript)
+if ($script:IsServiceGraph) {
+    # Build enable metrics filter Helm set
```
Comment (mirii1994) [inline datasources/windows/kubernetes/digitalocean/kubernetes/telemetry/traces/traces.ps1:32]: This comment is a copy-paste mistake? it supposed to be "build enable service graph"?

File: datasources/windows/kubernetes/eks/kubernetes/telemetry/traces/traces.ps1 [hunk 7]
```diff
@@ -26,6 +26,11 @@ Invoke-Task 'Get-LogzioSpmToken' @{} 'Getting Logz.io SPM token' @($TracesFuncti
 Invoke-Task 'Build-EnableSpmHelmSet' @{} 'Building enable SPM Helm set' @($TracesFunctionsScript)
 # Build Logz.io SPM token Helm set
 Invoke-Task 'Build-LogzioSpmTokenHelmSet' @{SpmToken = $script:SpmToken} 'Building Logz.io SPM token Helm set' @($TracesFunctionsScript)
-
+# Get if service graph option was selected
+Invoke-Task 'Get-IsServiceGraphWasSelected' @{MetricsParams = $script:MetricsParams} 'Getting if service graph was selected' @($MetricsFunctionsScript)
+if ($script:IsServiceGraph) {
+    # Build enable metrics filter Helm set
```
Comment (mirii1994) [inline datasources/windows/kubernetes/eks/kubernetes/telemetry/traces/traces.ps1:32]: This comment is a copy-paste mistake? it supposed to be "build enable service graph"?

File: datasources/windows/kubernetes/gke/kubernetes/telemetry/traces/traces.ps1 [hunk 8]
```diff
@@ -26,6 +26,11 @@ Invoke-Task 'Get-LogzioSpmToken' @{} 'Getting Logz.io SPM token' @($TracesFuncti
 Invoke-Task 'Build-EnableSpmHelmSet' @{} 'Building enable SPM Helm set' @($TracesFunctionsScript)
 # Build Logz.io SPM token Helm set
 Invoke-Task 'Build-LogzioSpmTokenHelmSet' @{SpmToken = $script:SpmToken} 'Building Logz.io SPM token Helm set' @($TracesFunctionsScript)
-
+# Get if service graph option was selected
+Invoke-Task 'Get-IsServiceGraphWasSelected' @{MetricsParams = $script:MetricsParams} 'Getting if service graph was selected' @($MetricsFunctionsScript)
+if ($script:IsServiceGraph) {
+    # Build enable metrics filter Helm set
```
Comment (mirii1994) [inline datasources/windows/kubernetes/gke/kubernetes/telemetry/traces/traces.ps1:32]: This comment is a copy-paste mistake? it supposed to be "build enable service graph"?

File: resources-linux/k8s/datasource_traces_utils.bash [hunk 9]
```diff
@@ -124,3 +124,60 @@ function build_logzio_spm_token_helm_set {
...
+    send_log_to_logzio "$LOG_LEVEL_DEBUG" "$message" "$LOG_STEP_TRACES" "$LOG_SCRIPT_TRACES" "$func_name" "$AGENT_ID" "$PLATFORM" "$SUB_TYPE" "$CURRENT_DATA_SOURCE"
+    write_log "$LOG_LEVEL_DEBUG" "$message"
+
+    write_task_post_run "IS_SERVICE_GRAPH=$is_servicegraph"
+}
+# Builds enable service graph Helm set
+# Input:
+#   ---
+# Output:
+#   LOG_HELM_SETS - Contains all the Helm sets for logging
```
Comment (mirii1994) [inline resources-linux/k8s/datasource_traces_utils.bash:166]: for logging?

**Review by mirii1994 — CHANGES_REQUESTED** [review]

File: resources-linux/k8s/datasource_traces_utils.bash (hunk 9 above)
Comment (ralongit) [reply resources-linux/k8s/datasource_traces_utils.bash:166]: Yes, internal logging. Please note it's like this across all of the scripts.

**Comment by ralongit** [comment]
Will not be implemented, it will be part of the SPM flag.

**Pull request closed without merging**